process/transactionLog/process.go

```diff
@@ -93,6 +124,12 @@ func (tlp *txLogProcessor) SaveLog(txHash []byte, tx data.TransactionHandler, lo
 		})
 	}
 
+	if tlp.storeLogsInRAM {
+		tlp.mut.Lock()
```

Comment:
make a small function with this if and save.

Reply:
done.